docs/getting-started.mdx

```diff
@@ -121,6 +121,12 @@ try {
 Sequelize will keep the connection open by default, and use the same connection for all queries.
 If you need to close the connection, call `sequelize.close()` (which is asynchronous and returns a Promise).
 
+:::note
+
+One `sequelize.close()` has been called, it's impossible to reopen a connection. You will need to create a new Sequelize instance to access your database again.
```

Comment:
Sorry that's on me, I had a typo in my previous suggestion 🤦🏼‍♀️ 

```suggestion
Once `sequelize.close()` has been called, it's impossible to open a new connection. You will need to create a new Sequelize instance to access your database again.
```

Reply:
Thanks